massgov/openmass
# DP-24290 Navigation labels

**Description:**
Apply the MF changes to the theming templates.

**Jira:** (Skip unless you are MA staff)
DP-24290


**To Test:**
[See the MF PR for details.](https://github.com/massgov/mayflower/pull/1743)


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
